source/plugins/code/metadata.yml

```diff
@@ -33,6 +33,14 @@ inputs:
     min: 100
     max: 1000
 
+  plugin_activity_days:
```

Comment:
shouldnt it be  `_code_`?

Reply:
Indeed, nice catch 👍 
Changed it in [a6932e9](https://github.com/lowlighter/metrics/commit/a6932e9d4a1f085c8fc9724dc68c9a8bb18bd1e1)